compose/install.md

```diff
@@ -5,6 +5,8 @@ title: Install Docker Compose
 toc_max: 2
 ---
 
+{% assign composeversion = '1.15.0' %}
```

Comment:
It would be cool if you moved these version definitions to the `_config.yml` file. That way, when  multiple products were released, you could go into a single file and update all the versions for the products.
With the definition being done on this file, you need to scavenger for the files that need to be updated on a release.